### PR TITLE
Do some cleanups and housekeeping

### PR DESCRIPTION
**Pull request by NoComment1105**
:D

### CODE REVIEW

File: libs.versions.toml [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 
 jansi = "2.4.0"
-kord-extensions = "1.5.5-20221107.093708-35"
+kord-extensions = "1.5.5-20221126.235557-39"
 kx-ser = "1.4.1"
 logging = "2.1.23" # DON'T REMOVE OR UPDATE
 logback = "1.2.8" # DON'T REMOVE OR UPDATE
 gson = "2.10"
-groovy = "3.0.8" # DON'T REMOVE OR UPDATE
+groovy = "3.0.10" # DON'T REMOVE OR UPDATE
```
Comment (NotJansel) [inline libs.versions.toml:10]: May I ask why you have changed the Version when specifically commented to not change these? This version change broke the logging for me last time.

File: src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt [hunk 2]
```diff
@@ -81,102 +83,82 @@ class Modrinth : Extension() {
...
+                                    embedContents(hit, strLoaders)
                                 }
                             }
-                        }
-                        timeoutSeconds = 60
-                        locale = Locale.ENGLISH
-                    }.send()
+                            timeoutSeconds = 300
+                            locale = Locale.ENGLISH
+                        }.send()
```
Comment (NotJansel) [inline src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt]: Also why the longer timeout?

**Review by NotJansel — COMMENTED** [review]
I have left some comments in the files changes, please explain further in these to my questions. Explicitily: why does the modrinth command need a longer timeout? And why did you change the groovy version from 3.0.8 to 3.0.10 when I explicitly commented to not change these?

File: libs.versions.toml (hunk 1 above)
Comment (NoComment1105) [reply libs.versions.toml:10]: Groovy plays no part in the issue, it's just the other two

File: src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt (hunk 2 above)
Comment (NoComment1105) [reply src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt]: Better accessibility for users, gives them more time to read. Can be reduced back down to 2 or 3 minutes as oppose to 5

Comment (NotJansel) [reply src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt]: I think 3 minutes would be enough

File: libs.versions.toml (hunk 1 above)
Comment (NotJansel) [reply libs.versions.toml:10]: I see

File: src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt (hunk 2 above)
Comment (NoComment1105) [reply src/main/kotlin/org/hyacinthbots/allium/extensions/Modrinth.kt]: Done

**Review by NotJansel — APPROVED** [review]
alright, I will merge this in a moment

**Pull request merged**